### PR TITLE
Created Company model object using Builder Pattern

### PR DESCRIPTION
**Pull request by obillaneni**
Hi Team,

Can you please review the code for CompanyModel object created.

Thanks
Vamsi

### CODE REVIEW

File: HRSolutions/src/main/java/org/hrSolution/model/CompanyRegistrationModel.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package org.hrSolution.model;
+
+import org.codehaus.jackson.annotate.JsonProperty;
+import org.hrSolution.model.PreEmployeeRegistrationModel.PreEmployeeRegistrationBuilder;
```
Comment (akb040193) [inline HRSolutions/src/main/java/org/hrSolution/model/CompanyRegistrationModel.java]: remove unused package

Comment (lalchandmali) [inline HRSolutions/src/main/java/org/hrSolution/model/CompanyRegistrationModel.java]: Remove unused import

File: HRSolutions/src/main/java/org/hrSolution/model/CompanyRegistrationModel.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+public class CompanyRegistrationModel {
+
+	@JsonProperty("cName")
```
Comment (akb040193) [inline HRSolutions/src/main/java/org/hrSolution/model/CompanyRegistrationModel.java:20]: Provide comment for each variable name

File: HRSolutions/src/main/java/org/hrSolution/model/CompanyRegistrationModel.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
+package org.hrSolution.model;
+
+import org.codehaus.jackson.annotate.JsonProperty;
+import org.hrSolution.model.PreEmployeeRegistrationModel.PreEmployeeRegistrationBuilder;
+
+public class CompanyRegistrationModel {
```
Comment (lalchandmali) [inline HRSolutions/src/main/java/org/hrSolution/model/CompanyRegistrationModel.java:11]: Please add more variables

File: HRSolutions/src/main/java/org/hrSolution/model/CompanyRegistrationModel.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+	@JsonProperty("cName")
+	private String companyName;
+	@JsonProperty("cDesc")
```
Comment (lalchandmali) [inline HRSolutions/src/main/java/org/hrSolution/model/CompanyRegistrationModel.java:26]: Please give one line space after each variable

File: HRSolutions/src/main/java/org/hrSolution/model/CompanyRegistrationModel.java [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+	public String getCompanyDescription() {
+		return companyDescription;
+	}
+
+	public static class CompanyRegistrationModelBuilder {
+
+		@JsonProperty("cName")
+		private String companyName;
+		@JsonProperty("cDesc")
```
Comment (lalchandmali) [inline HRSolutions/src/main/java/org/hrSolution/model/CompanyRegistrationModel.java:144]: Please give one line space after each variable decalaration

File: HRSolutions/src/main/java/org/hrSolution/model/CompanyRegistrationModel.java [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+	public String getCompanyName() {
+		return companyName;
+	}
+
+	public String getCompanyDescription() {
+		return companyDescription;
+	}
+
+	public static class CompanyRegistrationModelBuilder {
```
Comment (akb040193) [inline HRSolutions/src/main/java/org/hrSolution/model/CompanyRegistrationModel.java:133]: Write comment about this class as below:
Builder class of  CompanyRegistraionModel

**Comment by obillaneni** [comment]
Hello Team,

I have done changes to the code and pushed the latest one. Can you review again.

Thanks
Vamsi

**Comment by lalchandmali** [comment]
Well done

**Pull request merged**